packages/plugins/documentation/admin/src/pages/utils/useReactQuery.js

```diff
@@ -25,20 +39,26 @@ const useReactQuery = () => {
     });
   };
 
-  const deleteMutation = useMutation(deleteDoc, {
-    onSuccess: () => handleSuccess('info', 'notification.delete.success'),
-    onError: (error) => handleError(error),
-  });
+  const deleteMutation = useMutation(
+    ({ prefix, version }) => del(`${prefix}/deleteDoc/${version}`),
+    {
+      onSuccess: () => handleSuccess('info', 'notification.delete.success'),
+      onError: (error) => handleError(error),
```

Comment:
It is a very random thought but we should soon add to a todo list that we need to do this globally (on the client) in the next major version.